botcity/web/browsers/chrome.py

```diff
@@ -72,11 +70,10 @@ def default_options(headless=False, download_folder_path=None, user_data_dir=Non
             chrome_options.add_argument("--no-sandbox")
     except AttributeError:
         pass
-
     if not user_data_dir:
         temp_dir = tempfile.TemporaryDirectory(prefix="botcity_")
         user_data_dir = temp_dir.name
-        atexit.register(cleanup_temp_dir, temp_dir)
+        chrome_options._botcity_temp_dir = user_data_dir
```

Comment:
Indentation is wrong as we must always set this attribute.
```suggestion
    chrome_options._botcity_temp_dir = user_data_dir
```